opennms-base-assembly/src/main/filtered/etc/events/opennms.events.xml

```diff
@@ -914,13 +914,15 @@
                         &lt;p&gt;This event is generated when node outage processing
                         determines that the critical path IP address/service for
                         this node is not responding..&lt;/p&gt;
+                        &lt;p&gt;More information on the affected node(s) can be found at
+                        the &lt;a href="opennms/pathOutage/index.jsp"&gt;Path Outages&lt;/a&gt; page.&lt;/p&gt;
                 </descr>
     <logmsg dest="logndisplay">
                         %parm[nodelabel]% path outage. Critical path =
                         %parm[criticalPathIp]% %parm[criticalPathServiceName]%
                 </logmsg>
     <severity>Major</severity>
-    <alarm-data reduction-key="%uei%:%dpname%:%nodeid%" alarm-type="3" auto-clean="false"/>
+    <alarm-data reduction-key="%uei%:%parm[criticalPathIp]%:%parm[criticalPathServiceName]%" alarm-type="3" auto-clean="false"/>
```

Comment:
Ok, but in my system, ONMS 18.0.1 not using distributed pollers, I think %dpname% translate to the display name of the node affected by the path outage...

....

Ok, I may be wrong indeed....

I'll add %dpname% back